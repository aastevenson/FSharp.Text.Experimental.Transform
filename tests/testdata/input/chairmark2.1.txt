for i := 1 to 9 do
	for j := 1 to 10 do
		write i*j;
	end
end