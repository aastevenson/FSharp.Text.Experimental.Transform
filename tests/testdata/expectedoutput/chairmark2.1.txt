var i;
for i := 1 to 9 do
    var j;
    for j := 1 to 10 do
        write i * j;
    end
end
